vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -200,7 +200,8 @@ - (void)createWallet:(void (^)(NSError * _Nullable))completion
   //   - LEDGER_ERROR: Already initialized
   //   - BAD_REGISTRATION_RESPONSE: Request credentials call failure or malformed data
   //   - REGISTRATION_VERIFICATION_FAILED: Missing master user token
-  ledger->CreateWallet();
+  ledger->CreateWallet(^void(bool initializing){
```

Comment:
1. I don't understand the "initializing" arg. Isn't the callback called _after_ the wallet is _initialized_? Is the bool supposed to say whether or not it succeeded in initializing.

2. The callback for this is actually all the code above this that is currently set to `self.walletInitializedBlock` at the top of this method. Basically the entire contents of that assignment will be copy-pasted into this callback now. And then you can remove the `walletInitializedBlock` property along with its usages in `OnWalletInitialized`

I also assume `OnWalletInitialized` would also be removed from LedgerClient?